index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Week1-Assignment</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+    <!-- Navabr Start -->
+
+    <nav class="navbar_section">
+        <img src="images/logo.png" alt="logo">
+        <ul>
+            <div class="navbar">
+                <li>
+                    <a href="#home" class="active"> Home </a>
+                </li>
+                <li> <a href="#service"> Service </a>
+                </li>
+                <li>
+                    <a href="#about"> About </a>
+                </li>
+                <li>
+                    <a href="#contact"> Contact </a>
+                </li>
+                <li>
+                    <a href="#blog"> Blog </a>
+                </li>
+            </div><button>Log in</button>
+        </ul>
+    </nav>
+
+    <!-- Navbar end -->
+
+    <!-- Section1 start -->
+
+    <div class="section1_container">
+        <div class="section1_part1">
+            <p class="section1_top">#Get your 14 days Free trial</p>
+            <h1>Manage All Of Your Stuff Using A TGB</h1>
+            <p class="section1_bottom"> Lorem, ipsum dolor sit amet consectetur adipisicing elit. deserunt recusandae
+                amet omnis vel libero sequi perferendis?</p>
+            <div class="section1_imagepart">
+                <img src="images/logo.png" alt="">
+                <img src="images/logo.png" alt="">
+                <img src="images/logo.png" alt="">
+                <img src="images/logo.png" alt="">
+
+                <p>4m+ users</p>
+            </div>
+
+            <button>Learn More</button>
+        </div>
+
+        <div class="section1_part2">
+            <img src="images/iphone.jpg" alt="">
+        </div>
+    </div>
+    <!-- section1 end -->
+
+
+    <!-- section2 start -->
+    <div class="section2">
```

Comment:
Same here.
Don't use 1 or 2 in the naming. Instead specify the proper name like "body-container", "image-container" etc.